frontend/src/features/properties/list/IFlatProperty.ts

```diff
@@ -0,0 +1,69 @@
+import {
```

Comment:
This interface isn't currently used, but probably will need to be required for the Property List View.